WebRoot/skins/_base/base4/skin.css

```diff
@@ -758,6 +758,14 @@ we are hiding element with that class
     fill: @IconCheckboxUnchecked@;
 }
 
+.ZTreeItemCheckbox .ImgCheckboxChecked-svg .svg-icon {
```

Comment:
Was thinking not to have variables for them as we didn't had them for any of the svg icons (above & below this line). Tried to just refer to the color we have in a variable for the checkboxes. 

Let me know your thoughts.